mule-artifact-it/mule-deployer-it/src/test/java/integration/test/mojo/CloudHubDeploymentTest.java

```diff
@@ -176,8 +180,8 @@ public String getApplicationStatus() {
     @Override
     public Boolean run() {
       Application application = cloudHubClient.getApplication(applicationName);
-      applicationStatus = application.status;
-      if (application != null && expectedStatus.equals(application.status)) {
+      applicationStatus = application.getStatus();
+      if (application != null && expectedStatus.equals(application.getStatus())) {
```

Comment:
I don't get it. If application is null, isn't it to throw a NPE when invoking getStatus?